lib/restHttp/src/main/scala/com/gu/util/resthttp/RestRequestMaker.scala

```diff
@@ -29,8 +29,8 @@ object RestRequestMaker extends Logging {
       case success: JsSuccess[T] =>
         ClientSuccess(success.get)
       case error: JsError => {
-        logger.error(s"Failed to convert Zuora response to case case $error. Response body was: \n $bodyAsJson")
-        GenericError(s"Error when converting Zuora response to case class: $error")
+        logger.error(s"Failed to convert JSON response to case case $error. Response body was: \n $bodyAsJson")
```

Comment:
typo here, think it should be...
```suggestion
        logger.error(s"Failed to convert JSON response to case class $error. Response body was: \n $bodyAsJson")
```